osm.py

```diff
@@ -492,6 +538,7 @@ def main():
 
     try:
         vault_paths = get_vault_paths()
+        directives = list(get_processing_directives())
```

Comment:
I was on the fence about where in the code to do this as it is a defacto check on the configuration file even if no file operations are going to be performed.
On the one hand, validating early is good.
On the the other hand, validating when it won't be used (say for --list) could make it harder for someone to incrementally create/tweak their config...

Reply:
Either way is fine, of course - I can make UX arguments either way, and so for me, the UX arguments are kind of a wash.

Tiebreaker might be simplicity / readability of the code itself, and I think that argues for validating early.

Reply:
Yeah, I think I am ok with early too until a compelling argument comes in for postponing.